extensions/wikia/DesignSystem/DesignSystemHelper.class.php

```diff
@@ -47,17 +45,16 @@ public static function getSvg( $group, $name, $classNames = '', $alt = '' ) {
 	/**
 	 * @desc Loads SVG file as a SimpleXMLElement object or gets it from cache
 	 *
-	 * @param string $group
 	 * @param string $name
 	 *
 	 * @return SimpleXMLElement
 	 */
-	private static function getCachedSvg( $group, $name ) {
-		if ( isset( self::$svgCache[ $group ][ $name ] ) ) {
-			$xml = self::$svgCache[ $group ][ $name ];
+	public static function getCachedSvg( $name ) {
```

Comment:
Right. At some point this method was more complicated and I wanted to write a test. Now it makes sense to change it back to private.